controllers/articles.controller.js

```diff
@@ -15,10 +18,27 @@ function getArticleById(req, res, next) {
 }
 function getAllArticles(req, res, next) {
   selectAllArticles()
-    .then((articles) => {     
+    .then((articles) => {
       res.status(200).send({ articles })
     })
-    .catch(next)
+    .catch((err) => {
+      next(err)
+    })
+}
+function updateArticle(req, res, next) {
+  const { article_id } = req.params
+  const { inc_votes } = req.body
+  const promises = [updateSelectedArticle(article_id, inc_votes)]
+  if (article_id) {
```

Comment:
There will always be an article_id here for this controller to trigger! It could be that the article_id isn't found or is invalid, but it will always be there!